xclim/sdba/measures.py

```diff
@@ -496,6 +500,17 @@ def _taylordiagram(
         }
     )
 
+    # Normalize the standard deviations byt the standard deviation of the reference.
+    if normalize:
+        if (out[{"taylor_param": 0}] == 0).any():
+            raise ValueError(
+                "`ref_std =0` (homogeneous field) obtained, normalization is not possible."
+            )
+        out[{"taylor_param": [0, 1]}] = (
+            out[{"taylor_param": [0, 1]}] / out[{"taylor_param": 0}]
```

Comment:
Is throwing an error a good choice? 

If you send a homogenous field as a reference which will have `ref_std=0` , you should know you can't normalize this ... but technically, `ref_std=0` is computed inside the function, so you get a `ValueError` not from the input, something that's derived from the input within the function. I think it's all right, but I still want to point out this is a tricky ValueError that emerges from a computation inside the function

Reply:
My stats-fu is not strong, so I can't effectively weigh in on this. @aulemahal?